### ISSUE TITLE
TC, Chrome UX compatibility and scaling tweaks

### ISSUE THREAD

**Issue by apws** [mucommander/mucommander#437]
Hi all, I tried to tweak my experience with muCommander on linux and macos coming from my primary windows desktop and found differend set of key shortcuts more TC-like (with FEW linux/ubuntu shortcuts changes - that may be war...) - but its mostly for primary windwos users. On max its even different story with default modifiers (Fn, Ctrl, Alt, System) to be remapped rather system wide by Ukelele keyremapping app to follow standard (ya, it is possible, even mac settings changes to the modifiers IS useable somewhat, but this is NOT recognized by JAVA apps !!! - so here the Ukelele thing...). With this, almost unified experience can be achieved - I am still exploring some virtual mounts for linux volumes to mimic windows "Drive" concept to have persistent current folder. And some other muCommander shortcuts tweaks mimic more web browser interface (currently almost standard), so Ctrl+T for new tab, Ctrl+N for new window, Ctrl+D for bookmarks, ane more obvious to fill the gaps as Ctrl+F for FTP (and alikes) ... hope this helps to somebody.... it is really unified accros win/mac/lin. 

REQUIRED LINUX "dconf-editor" changes (most posible in ubuntu settings keyboard shortcuts too):

org.gnome.desktop.wm.keybindings

begin-move             Super+F9
begin-resize             Super+F8
panel-main-menu    Super+Escape
panel-run-dialog      Super+R
unmaximize              Super+Down (not possible by ubuntu settings?)

To even more unify experience and better handle scalling problems in Linux, set here also this tweak, java system wide (IntelliJ IDEA is okay with that)

sudo nano /etc/environment
add ... JAVA_TOOL_OPTIONS=-Dsun.java2d.uiScale=2

and use rather METAL look & feel everywhere, even on mac, as in linux, GTK+ handles fractional scaling together with gnome (say 125%) and it clashes with current state of scalling support in muCommander - metal simply handles better this java option
(having fonts at say 15pt bold everywhere in appearance settings, best seems to be Ubuntu font now)

**Comment by apws** [comment]
interesting that over wayland on linux the scaling works well even for GTK+ look, so no tweaks needed :)

**Comment by ahadas** [comment]
> interesting that over wayland on linux the scaling works well even for GTK+ look, so no tweaks needed :)

Yeah, you're right - maybe we should change the default on Linux to GTK+ then?

**Comment by ahadas** [comment]
As for the proposed keyboard shortcuts, yeah - it's debatable and that's why it was important to make it configurable.
For example, on Fedora CTRL+F takes you to the search widget in `Files` so having this shortcut mapped to `Find` in mucommander would make more sense to Fedora users than to edit-credentials
I think our way to go here is to provide shortcuts per operating system.

I didn't get your point about the virtual volumes - can you please clarify that?

**Comment by apws** [comment]
about volumes - its weird that mu isnt remembering "current folder" per each "drive/volume" ? or I am missing something ... I already peeked again into source to try to find something about that, and found till now setCurrentFolder and getVolumes methods - I see you do something smart about volumes per system - and there I will start on mac and linux to try to map/mount symlinks or so (I am not good in unix, though) ... but the ""current folder" remembered per drive/volume - is this handled? I heavily use this in TC, having more substed drives and each persists current folder when activating into view ... also activating identical srive/volume in other panel gets current folder of it too ... its useable very often to ZIP folder along etc...

**Comment by ahadas** [comment]
Ah I see what you mean.
So if you browsed to `c:` and entered folder `A`, then switched to drive `d:` and then switch back to drive `c:` you expect to land in folder `A`, right?
I wouldn't call it "current folder" but rather the last place you visited on that "drive".

You can change `CustomOpenLocationAction` in `DrivePopupButton` so when the user choose a drive, this action would check the last place that was visited on that drive (see `RecentLocationsQL`).

**Comment by Pasqualle** [comment]
@apws thanks for the keymap. I do not have to search for the right keys any more.
The actions which you use with BACK_SLASH key, do not seems to work in TC. Are those your custom shortcuts?
The alt+f7 is already used by Ubuntu, but [can be disabled](https://askubuntu.com/questions/75097/how-to-remove-altf7-window-moving-hot-key-in-xubuntu).

**Comment by pskowronek** [comment]
Is it still an issue?
